docs/en/guides/webhooks.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: Webhooks (Coming soon!)
+pagename: shotgun-webhooks
+lang: en
+---
+
+# Webhooks
+
+{% include info title="Beta" content="Webhooks is currently in beta. If you would like to join the beta, please email: [webhooks-beta@shotgunsoftware.com](mailto:webhooks-beta@shotgunsoftware.com)" %}
+
+Webhooks allow a service you control to be notified of events that occur in Shotgun. When you create a webhook, you specify the type of event you are interested in and tell Shotgun what URL to send data to when it is triggered. Once the relevant event happens in Shotgun, a payload of data describing the event will be sent to the webhook’s URL. This allows you to build tight integrations with Shotgun and automate portions of your workflow.
+
+## What are some examples of how to use webhooks?
+
+There are numerous use cases for webhooks. A few that we think are compelling are outlined in this document, but their uses are by no means limited to these few examples.
+
+### Create a directory structure on disk when an entity is created
```

Comment:
I specifically called this out as not a good idea here:
https://community.shotgunsoftware.com/t/creating-and-unregistering-folders-with-the-event-daemon/46
Am I wrong to suggest this? Folder creation on entity creation seems error prone.

Reply:
That's a question for @thebeeland – I've gone ahead and addressed the cosmetic fixes on this doc, but content-wise, I defer to him.